modules/serverless-creating-broker-labelling.adoc

```diff
@@ -2,19 +2,19 @@
 //
 // * serverless/event_workflows/serverless-using-brokers.adoc
 
-[id="serverless-creating-broker-admin_{context}"]
-= Creating a broker by labeling a namespace
+[id="serverless-creating-broker-labelling_{context}"]
+= Creating a broker by labelling a namespace
```

Comment:
IBM Style Guide recommends using _labeled_ over _labelled_. I reckon the same applies for _labeling_.